pisa/stages/flux/pi_mceq_barr.py

```diff
@@ -25,6 +25,21 @@
 from pisa.utils.resources import find_resource
 
 
+def eval_spline(true_log_energy, true_abs_coszen, spline):
```

Comment:
This looks like a redundant function; please remove, unless there's a good reason for it that I am missing.

Reply:
Function removed. I think this function used to do more than this but now it is a one-liner yes can just directly call that one line in the code.